elements/lisk-p2p/src/peer.ts

```diff
@@ -294,6 +315,13 @@ export class Peer extends EventEmitter {
 		};
 	}
 
+	public applyPenalty(penalty: number): void {
+		this._reputation -= penalty;
```

Comment:
what happens if the peer is already banned?

Reply:
If peer is already banned it would not exist in `peer_pool`. The `updatePeerScore` checks for this.